tests/support/Mock/TestImplementation.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
as I described above - we need to test that binded request class returns correct rules and you can use already implemented `TestRequest` class, so this class is useless and should be removed